covid-vue-app/src/components/DataTable.vue

```diff
@@ -227,6 +218,466 @@ export default {
+          /* eslint-disable no-console */
+          console.log(e);
+          /* eslint-enable no-console */
+        });
+    },
+    getDataforFrance() {
+      this.items[5].deaths = "Loading...";
+      this.items[5].recovered = "Loading...";
+      this.items[5].confirmed = "Loading...";
+      axios
+        .get(
+          "https://covid-19-coronavirus-statistics.p.rapidapi.com/v1/total",
+          {
+            params: {
+              country: "France",
+            },
+            headers: {
+              "x-rapidapi-key": process.env.VUE_APP_COVIDAPIKEY,
+              "x-rapidapi-host":
+                "covid-19-coronavirus-statistics.p.rapidapi.com",
+            },
+          }
+        )
+        .then((response) => {
+          if(response.data.data.deaths == null){
+            this.items[5].deaths = "N/A";
+          }
+          else{
+            this.items[5].deaths = response.data.data.deaths;
+          }
+
+          if(response.data.data.recovered == null){
+            this.items[5].recovered = "N/A";
+          }
+          else{
+            this.items[5].recovered = response.data.data.recovered;
+          }
+
+          if(response.data.data.confirmed == null){
+            this.items[5].confirmed = "N/A";
+          }
+          else{
+            this.items[5].confirmed = response.data.data.confirmed;
+          }
+        })
+        .catch((e) => {
+          /* eslint-disable no-console */
+          console.log(e);
+          /* eslint-enable no-console */
+        });
+    },
+    getDataforSpain() {
+      this.items[6].deaths = "Loading...";
+      this.items[6].recovered = "Loading...";
+      this.items[6].confirmed = "Loading...";
+      axios
+        .get(
+          "https://covid-19-coronavirus-statistics.p.rapidapi.com/v1/total",
+          {
+            params: {
+              country: "Spain",
+            },
+            headers: {
+              "x-rapidapi-key": process.env.VUE_APP_COVIDAPIKEY,
+              "x-rapidapi-host":
+                "covid-19-coronavirus-statistics.p.rapidapi.com",
+            },
+          }
+        )
+        .then((response) => {
+          if(response.data.data.deaths == null){
+            this.items[6].deaths = "N/A";
+          }
+          else{
+            this.items[6].deaths = response.data.data.deaths;
+          }
+
+          if(response.data.data.recovered == null){
+            this.items[6].recovered = "N/A";
+          }
+          else{
+            this.items[6].recovered = response.data.data.recovered;
+          }
+
+          if(response.data.data.confirmed == null){
+            this.items[6].confirmed = "N/A";
+          }
+          else{
+            this.items[6].confirmed = response.data.data.confirmed;
+          }
+        })
+        .catch((e) => {
+          /* eslint-disable no-console */
+          console.log(e);
+          /* eslint-enable no-console */
+        });
+    },
+    getDataforItaly() {
```

Comment:
Perhaps this could be abstracted since there is quite a few repeating code.

Maybe

- Make a function with country name as a parameter
- Start them in an array, and perhaps use map or const instead of addressing them by `this.item[<some number>].<something>`